src/server/scripts/Draenor/zone_assault_on_the_dark_portal.cpp

```diff
@@ -61,8 +69,57 @@ class quest_the_cost_of_war : public QuestScript
     }
 };
 
+// 34422 - Blaze of Glory
+class quest_blaze_of_glory : public QuestScript
+{
+public:
+    quest_blaze_of_glory() : QuestScript("quest_blaze_of_glory") { }
+
+    void OnQuestStatusChange(Player* player, Quest const* /*quest*/, QuestStatus /*oldStatus*/, QuestStatus newStatus) override
+    {
+        if (newStatus == QUEST_STATUS_NONE)
+        {
+            player->RemoveAurasDueToSpell(SPELL_BLEEDING_HOLLOW_HOLDOUT);
+            player->RemoveAurasDueToSpell(SPELL_BLEEDING_HOLLOW_TRAIL_OF_FLAME);
+            PhasingHandler::OnConditionChange(player);
+        }
+        else if (newStatus == QUEST_STATUS_INCOMPLETE)
+        {
+            if (player->GetQuestStatus(QUEST_FLAG_ARMY_PUSHED) != QUEST_STATUS_REWARDED)
+            {
+                player->CastSpell(player, SPELL_BLEEDING_HOLLOW_HOLDOUT, TRIGGERED_FULL_MASK);
+                player->CastSpell(player, SPELL_BLEEDING_HOLLOW_TRAIL_OF_FLAME, TRIGGERED_FULL_MASK);
+                PhasingHandler::OnConditionChange(player);
+            }
+            else
+            {
+                player->CastSpell(player, SPELL_BLEEDING_HOLLOW_TRAIL_OF_FLAME, TRIGGERED_FULL_MASK);
```

Comment:
you can put these 2 lines outside the else body, as its used in both cases (you can remove these 2 lines in if body then)